lib/streamlit/watcher/local_sources_watcher.py

```diff
@@ -81,7 +86,11 @@ def close(self):
         self._is_closed = True
 
     def _register_watcher(self, filepath, module_name):
+        global FileWatcher
         if FileWatcher is None:
+            FileWatcher = get_default_file_watcher_class()
+
+        if FileWatcher is NoOpFileWatcher:
             return
```

Comment:
`return None`?

Reply:
I think it's cleanest to keep it just `return` for now since this method is run purely for its side effect. Also, this keeps consistency with the rest of the class' methods that are run only for their side effect (none of them explicitly return `None`)